resources/js/app/components/placeholder-list/placeholder-params.vue

```diff
@@ -52,6 +52,7 @@
 </template>
 <script>
 import { EventBus } from 'app/components/event-bus';
+import { placeholderParamsDecoded, placeholderParamsEncode } from 'app/helpers/placeholder-encoding';
```

Comment:
`atob` stands for `ascii to binary conversion` and `btoa` stands for `binary to ascii conversion`.
We could rename these new functions to something like `asciiToBinaryUtf8` and `binaryToAsciiUtf8`. Or the functions could be `encode***` and `decode***`.

Reply:
If we decide to change these names we can move all methods to text helpers. I will to do

Reply:
If we decide to change these names we can move all methods to text helpers. I will to do